### PR TITLE
Use prod ccdb

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
Let's see if this works. There is a bunch of tests from the detectors which also use the test instance, but hopefully they are not part of the CI.

**Comment by jgrosseo** [comment]
Excellent! And let's extend all the validity of the accessed objects until 2050, so we are good...

File: CCDB/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ std::map<std::string, std::string> headers = f.api.retrieveHeaders("FOO/DeadPixe
 // init
 CcdbApi api;
 map<string, string> metadata; // can be empty
-api.init("http://ccdb-test.cern.ch:8080"); // or http://localhost:8080 for a local installation
+api.init("http://alice-ccdb.cern.ch:8080"); // or http://localhost:8080 for a local installation
```
Comment (jgrosseo) [inline CCDB/README.md]: Drop port

Comment (jgrosseo) [reply CCDB/README.md]: Everywhere :)

Comment (ktf) [reply CCDB/README.md]: Hopefully fixed.

**Pull request closed without merging**